.github/workflows/release.yml

```diff
@@ -70,6 +70,7 @@ jobs:
         GOOGLE_CLIENT_ID: ${{ secrets.GOOGLE_CLIENT_ID }}
         GOOGLE_CLIENT_SECRET: ${{ secrets.GOOGLE_CLIENT_SECRET }}
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NODE_OPTIONS: --max-old-space-size=4096
```

Comment:
Did you have any problems?

Reply:
Yes. The release crashed on out of memory

Reply:
Interesting, what has changed?

Reply:
Don't know... But the mac release is relatively new and oom issues can happen inconsistently if it gets close to the limit

Reply:
Oh, Mac release, OK